tock/tock/middleware.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime, timedelta
+from django.conf import settings
+from django.contrib import auth
+
+
+class AutoLogout(object):
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+
+        # Check if user exists and is logged in
+        if request.user and request.user.is_authenticated():
+
+            logout_time_in_seconds = settings.AUTO_LOGOUT_DELAY_MINUTES * 60
+            fmt = '%Y%m%d%H%M%S'
```

Comment:
You need to move this up and out of the `if` block, or L34 fails. That looks like what's making Circle unhappy.